clients/lcdexec/specification/lcdexec-spec.ini

```diff
@@ -1,7 +1,7 @@
 []
-mountpoint = lcdexec.conf
-;infos/plugins = dump validation enum range ipaddr path
-infos/plugin = dump validation enum
+mountpoint = lcdexec-spec.conf
```

Comment:
lcdexec.conf was okay, the name of the file itself (the spec) and the file to be used as configuration should be different to make it less confusing.